pyproject.toml

```diff
@@ -31,6 +31,7 @@ schema = "0.7.5"
 cfn-flip = "1.3.0"
 moto = {extras = ["all"], version = "^4.1.12"}
 aiohttp = "^3.8.4"
+certifi = "^2023.07.22"
```

Comment:
Why is this added? It doesn't seem relevant to the other changes to me.

Reply:
Ah, I may have connected that if I was more intelligent, but as I'm not, maybe explicitly mentioning `certifi` in the description would have saved me starting this thread.